libs/@local/hash-graph-client/package.json

```diff
@@ -17,9 +16,11 @@
     "@apps/hash-graph": "0.0.0-private",
     "@local/eslint-config": "0.0.0-private",
     "@local/tsconfig": "0.0.0-private",
+    "@openapitools/openapi-generator-cli": "2.5.2",
```

Comment:
We do have this as a dev-dependency, would it be required as a dependency when doing a "production only" yarn install in dockerfiles? I think it would fail in those cases (i.e. for the hash-api dockerfile)

Reply:
Good point, I made it a normal dependency.